src/api-v1.js

```diff
@@ -580,6 +580,105 @@ api.declare({
 
 api.declare({
   method: 'put',
+  route: '/ami-set/:amiSetId',
+  name: 'createAmiSet',
+  deferAuth: true,
+  scopes: [['aws-provisioner:manage-ami-set:<amiSetId>']],
+  input: 'create-ami-set-request.json#',
+  output: 'get-ami-set-response.json#',
+  title: 'Create new AMI Set',
+  stability:  base.API.stability.stable,
+  description: [
+    'Create an AMI Set. An AMI Set is a collection of AMIs with a single name.',
+  ].join('\n'),
+}, async function (req, res) {
+  let input = req.body;
+  let amiSet = req.params.amiSetId;
```

Comment:
I probably would call this `amiSetId` since you'll be creating an Entity and probably want this name for that, instead of `aSet`